jest.config.js

```diff
@@ -69,7 +69,14 @@ module.exports = {
 
   // A set of global variables that need to be available in all test environments
 
-  // globals: {},
+  globals: {
+    'ts-jest': {
+      tsconfig: {
+        allowJs: true,
+        sourceMap: true,
```

Comment:
This line fixes the bug that came with `jest` v27 that it shows `import` and empty lines as uncovered also showing wrong line for failed test.